iree/compiler/Dialect/Vulkan/Utils/test/target_env_conversion.mlir

```diff
@@ -1,5 +1,5 @@
-// RUN: iree-opt -pass-pipeline='iree-hal-transformation-pipeline{serialize-executables=false}' -iree-hal-target-backends=vulkan-spirv %s | IreeFileCheck %s -check-prefix=DEFAULT
-// RUN: iree-opt -pass-pipeline='iree-hal-transformation-pipeline{serialize-executables=false}' -iree-hal-target-backends=vulkan-spirv -iree-vulkan-target-env='#vk.target_env<v1.0, r(10), [VK_KHR_shader_float16_int8, VK_KHR_storage_buffer_storage_class], {maxComputeWorkGroupInvocations = 64: i32, maxComputeWorkGroupSize = dense<[8, 8, 8]>: vector<3xi32>, shaderFloat64, shaderInt8}>' %s | IreeFileCheck %s -check-prefix=V10
+// RUN: [[ $IREE_VULKAN_DISABLE == 1 ]] || (iree-opt -pass-pipeline='iree-hal-transformation-pipeline{serialize-executables=false}' -iree-hal-target-backends=vulkan-spirv %s | IreeFileCheck %s -check-prefix=DEFAULT)
```

Comment:
Can we add an early return to iree/compiler/Dialect/Vulkan instead? Then we keep IREE_VULKAN_DISABLE only for when the driver isn't available.

Reply:
Not as easily, no. `iree::compiler::Dialect::Vulkan::IR` is used in all sorts of places like

https://github.com/google/iree/blob/3863d14efd58477afbc068317d989c0d579084c8/iree/tools/CMakeLists.txt#L145

Reply:
Maybe the test then? `iree/compiler/Dialect/Vulkan/Utils/test/CMakeLists.txt`

Reply:
Good idea, conditionally disabled in the test file and reverted the change to the .mlir.